tests/testthat/reporters/check.txt

```diff
@@ -19,40 +19,40 @@ Error: stop
 Error: !
 Backtrace:
     █
- 1. └─f()
- 2.   └─g()
- 3.     └─h()
+ 1. └─f() reporters/tests.R:47:2
```

Comment:
I'm not sure why this now generates srcrefs. But I'm pretty sure it's just an artefact of the testing setup of testthat; it shouldn't affect other packages.